### DISCUSSION TITLE
Dynamic purchasing system - Competition Result

### DISCUSSION THREAD

**Discussion by DeanFirkelj** [OP-TED/eForms-SDK#599] [Q&A] [answered]
Hi,
In case of DPS and Competition Results, trying to understand what scenario is supported:
1. For each competition instance under DPS, a contract is signed. Each of those contracts is sent as eForm to TED as Competition Result. When the DPS is terminated (as a separate event), it's rendered only as DPSTerminationIndicator -> True without any of lot results / settled contracts rendered (since all of those information was already transmitted)
2. Only upon the termination of the DPS (LotResult / DPSTerminationIndicator -> True) a Competition result is being formed with the list of all contracts settled during the whole duration of DPS and rendered in the one and only notice, triggered exclusively by the DPS termination.

Thnx,
Dean Firkelj

**Comment by YvesJo** [comment, marked as answer]
Hi,
Following D24 Art 50(3):

> Contracting authorities shall send a contract award notice within 30 days after the award of each contract based on a dynamic purchasing system. They may, however, group such notices on a quarterly basis. In that case, they shall send the grouped notices within 30 days of the end of each quarter.

so the last option is to be rejected. The indicator should be set to true with the last set of awarded contracts, should the DPS be terminated earlier.
For non-technical question please get in touch directly with the Helpdesk [ted@publications.europa.eu](mailto:ted@publications.europa.eu).
